tests/Neo.UnitTests/Cryptography/UT_Cryptography_Helper.cs

```diff
@@ -55,13 +57,21 @@ public void TestMurmurReadOnlySpan()
             input.Murmur128(0).Should().Equal(input2.Murmur128(0));
         }
 
+        [TestMethod]
+        public void TestXxHash3()
+        {
+            byte[] data = Encoding.ASCII.GetBytes(string.Concat(Enumerable.Repeat("Hello, World!^_^", 16 * 1024)));
+            data.XxHash3_32().Should().Be(HashCode.Combine(XxHash3.HashToUInt64(data, 40343)));
+            XxHash32.HashToUInt32(data).Should().Be(682967318u);
```

Comment:
Why this test, and why don't use `XxHash32.HashToUInt32` instead of `XxHash3.HashToUInt64` for hashcode?

Reply:
> Why this test, and why don't use `XxHash32.HashToUInt32` instead of `XxHash3.HashToUInt64` for hashcode?

Left over from the modification.